python/CHANGELOG.txt

```diff
@@ -0,0 +1,458 @@
+
```

Comment:
You should not be using:

```
git add -A
```

or 

```
git add .
```

This is an incorrect file added in an incorrect location.  Please remove it via:

```
cd $GEM_BASE
git rm python/CHANGELOG.txt
```

Please only add files in a targeted way, e.g.:

```
git add <filename>
```